docs/installation/windows.md

```diff
@@ -31,7 +31,8 @@ run the Docker daemon. The VirtualBox VM runs completely from RAM, is a small
 
 ## Requirements
 
-Your machine must be running Windows 7 or newer to run Docker. To find out what version of Windows you have:
+Your machine must be running Windows 7 or newer to run Docker. In addition, you must have a 64-bit operating system (x64).
+To find out what version of Windows you have:
```

Comment:
@aditirajagopal Good idea to summarize the requirement and then show how to get them. That means tweaking the first paragraph a bit.

```markdown
To run Docker, your machine must have a 64-bit operating system running Windows 7 or higher. Additionally, you must make sure that virtualization is enabled on your machine. To verify your machine meets these requirements, do the following:
```